website/docs/style/icon/icons-list.js

```diff
@@ -4328,6 +4476,22 @@ const iconsList = {
       group: 'Color',
       tags: ['hubspot', 'хабспот', 'logo', 'лого'],
     },
+    // {
+    //   name: 'InstagramColored',
+    //   size: ['l', 'm'],
+    //   group: 'Color',
+    //   tags: [
+    //     'instagram',
+    //     'social',
+    //     'logo',
+    //     'facebook',
```

Comment:
if we added "facebook" here, maybe also add "meta"?

Reply:
Good point! I'll add